### ISSUE TITLE
Add conferenceData options

### ISSUE THREAD

**Issue by OchotaDariusz** [Kubessandra/react-google-calendar-api#97]
That would be nice to have possibility to generate also i.e. google meet link attached to event

**Comment by Kubessandra** [comment]
Yes, I a agree I don't have a lot of time to work on this nowadays, if you want to make a quick PR I can review it
